nix/machines/core/common.nix

```diff
@@ -23,6 +23,8 @@
     bind
     kea
     scaleInventory
+    vim
```

Comment:
Finally!

Reply:
You can half my payments.

if only you could divided by zero?